libraries/toolbar/src/main/java/com/trendyol/uicomponents/toolbar/ToolbarViewState.kt

```diff
@@ -19,7 +19,14 @@ data class ToolbarViewState(
     @StyleRes val middleTextAppearance: Int = R.style.Trendyol_UIComponents_Toolbar_Text_Title,
     @StyleRes val upperRightTextAppearance: Int = R.style.Trendyol_UIComponents_Toolbar_Text_UpperAction,
     @StyleRes val lowerRightTextAppearance: Int = R.style.Trendyol_UIComponents_Toolbar_Text_LowerAction,
-    @DrawableRes val toolbarBackground: Int = android.R.color.white
+    @DrawableRes val toolbarBackground: Int = android.R.color.white,
+    val upperLeftTextMarginStartInPixel: Int? = null,
+    val lowerLeftTextMarginStartInPixel: Int?= null,
+    val upperRightTextMarginEndInPixel: Int?= null,
+    val lowerRightTextMarginEndInPixel: Int?= null,
+    val rightImageDrawableMarginEndInPixel: Int?= null,
+    val leftImageDrawableMarginStartInPixel: Int?= null
+
```

Comment:
```suggestion
    val upperLeftTextMarginStartInPixel: Int = 0,
    val lowerLeftTextMarginStartInPixel: Int = 0,
    val upperRightTextMarginEndInPixel: Int = 0,
    val lowerRightTextMarginEndInPixel: Int = 0,
    val rightImageDrawableMarginEndInPixel: Int = 0,
    val leftImageDrawableMarginStartInPixel: Int = 0
```

Reply:
When these are set as null it prevents binding adapter to set 0 for margin values